optaplanner-integration/optaplanner-spring-boot-autoconfigure/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.optaplanner</groupId>
+    <artifactId>optaplanner-integration</artifactId>
+    <version>7.32.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>optaplanner-spring-boot-autoconfigure</artifactId>
+
+  <name>OptaPlanner Spring Boot autoconfigure</name>
+  <description>
+    OptaPlanner solves planning problems.
+    This lightweight, embeddable planning engine implements powerful and scalable algorithms
+    to optimize business resource scheduling and planning.
+
+    This module contains the Spring Boot autoconfigure.
+  </description>
+  <url>https://www.optaplanner.org</url>
+
+  <dependencies>
+    <!-- Internal dependencies -->
+    <dependency>
+      <groupId>org.optaplanner</groupId>
+      <artifactId>optaplanner-core</artifactId>
+    </dependency>
```

Comment:
Should this be optional, I believe. IIUC, in theory, you can have optaplanner-autoconfiguration and unless you also have optaplanner-core on classpath, this autoconfiguration can be skipped. Conversely, if you hard-depend on optaplanner-core here, it will always be on classpath and so this autoconfiguration can never be skipped.

https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-custom-starter-module-autoconfigure:
> You should mark the dependencies to the library as optional so that you can include the autoconfigure module in your projects more easily. If you do it that way, the library is not provided and, by default, Spring Boot backs off.